gtfs_realtime_translators/translators/mnmt.py

```diff
@@ -0,0 +1,81 @@
+import json
+
+import pendulum
+
+from gtfs_realtime_translators.factories import TripUpdate, FeedMessage
+
+
+class MnmtGtfsRealtimeTranslator:
+    TIMEZONE = 'America/Chicago'
+
+    def __call__(self, data):
+        json_data = json.loads(data)
+
+        stops_list = json_data.get('stops')
+        departures_list = json_data.get('departures')
+
+        entities = []
+        if stops_list and departures_list:
+            entities = self.__make_trip_updates(stops_list, departures_list)
+
+        return FeedMessage.create(entities=entities)
+
+    @classmethod
+    def __make_trip_updates(cls, stops_list, departures_list):
```

Comment:
What does cls stand for?

Reply:
Class. Class methods require the first argument to be `cls`. Instance methods require the first argument to be `self`.

https://docs.python.org/3/library/functions.html#classmethod

Reply:
@joshverma Gotcha, thanks for sharing.